.env

```diff
@@ -54,7 +54,7 @@ UBUNTU=20.04
 
 # Default versions for various dependencies
 CLANG_TOOLS=14
-CUDA=11.0.3
+CUDA=11.4.2
```

Comment:
We can perhaps revert this change now?

Reply:
As the cuda version is hardcoded we can change it